distributed/stealing.py

```diff
@@ -134,8 +133,10 @@ def steal_time_ratio(self, ts):
         if cost_multiplier > 100:
             return None, None
 
-        level = int(round(log(cost_multiplier) / log_2 + 6, 0))
-        level = max(1, level)
+        level = int(round(log2(cost_multiplier) + 6))
```

Comment:
`log2` was added in Python 3.3+ and is faster than hand rolling by 2x

```python
In [3]: %timeit log(3) / log(2)
401 ns ± 5.33 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [4]: %timeit log(3, 2)
234 ns ± 2.72 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

`round` already rounds to `0` digits after the decimal place. Using the default is 2x faster than specifying one

```python

In [7]: %timeit round(1.5, 0)
393 ns ± 2.58 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [8]: %timeit round(1.5)
195 ns ± 1.77 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```